src/main/java/frc/robot/RobotContainer.java

```diff
@@ -45,19 +47,25 @@ public class RobotContainer {
     Controllers controllers;
     public AutoDriveBuilder autoDriveBuilder;
     public AutoShootBuilder autoShootBuilder;
-
+    public LED espSubsystem; 
     Command floorIntakeCommand;
     Command outtakeCommand;
     Command hpIntakeCommand;
+    Command ledCommand; 
+    Command testCommand; 
+    Command analogCommand; 
+    
 
     public RobotContainer() {
         swerveSubsystem = new Swerve();
         shooterSubsystem = new Shooter();
         pivotSubsystem = new Pivot(shooterSubsystem);
         climberSubsystem = new Climber();
         intakeSubsystem = new Intake();
+        espSubsystem = new LED(); 
         controllers = Controllers.getInstance();
-
+        ledCommand = new LEDCommand(shooterSubsystem, espSubsystem); 
```

Comment:
to reduce complexity of stuff stored everywhere, it is probably better to instantiate it directly into setDefaultCommand

Reply:
Changed